clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/handlers/BasicGoogleDeployHandler.groovy

```diff
@@ -534,4 +549,21 @@ class BasicGoogleDeployHandler implements DeployHandler<BasicGoogleDeployDescrip
       null
     }
   }
+
+  Map getUserData(BasicGoogleDeployDescription description, String serverGroupName,
+                  String instanceTemplateName, GoogleNamedAccountCredentials credentials) {
+    String customUserData = ''
+    if (description.userDataType && description.userData) {
+      if (description.userDataType.equals("URL")) {
```

Comment:
@duftler I don't think we care about the `URL` type. We will be putting a template file on the servers. I will double check when I get into the office though.